addOns/webuipoc/README.md

```diff
@@ -35,16 +35,22 @@ We would still like you to contribute a basic PoC but we are very happy for you
 
 ## Creating Your PoC
 
-Create a new directory for your add-on under [webuipoc](src/main/zapHomeFiles/webuipoc).
+Create a new directory for your add-on under [src/main/pocs](src/main/pocs).
 
 This is where you should put all of the files that need to be served up by ZAP.
 
 You may well find it easier to edit these files in the 'webuipoc' directory used by the ZAP runtime, this will be under
 the ZAP [default directory](https://www.zaproxy.org/faq/what-is-the-default-directory-that-zap-uses/).
-Once everything is working how you want you can copy them back to your new directory under 
-[webuipoc](src/main/zapHomeFiles/webuipoc) for your pull request.
-
-Source TypeScript files should be added to a new directory under the [src](src) directory.
+Once everything is working how you want you can copy them back to your new directory under
+[src/main/pocs](src/main/pocs) for your pull request.
+
+If your PoC uses React or some other JS framework, then it must follow the below guidelines so that the build files
```

Comment:
Swap the order on "below guidelines" ➡️ "guidelines below"

Reply:
That's the only thing I see but you and @thc202 probably know/understand the build better than I do.